etc/syntax/reference-files/suite_jinja2.rc

```diff
@@ -0,0 +1,70 @@
+#!Jinja2
+# ^ SYNTAX:3.1
+
+# SYNTAX numbering matches the table in https://github.com/cylc/cylc/issues/2752
+
+# SYNTAX: 3.2
+# templating code should be displayed diffently to cylc code
+{# no comment #}  # should display as a comment
+{% set foo = "bar" %}
+{% set foo = '%s' % 'bar' %}
+{% set foo = {"1": 2, "3", []} %}
+{% set foo = foo.bar(1, 2, 3) %}
+{{ foo_bar }}
+{{ foo.bar(1, 2, 3) }}
+{{ foo | bar(1, 2, 3) }}
+{% include 'foo' %}
+{% include ['foo', 'bar'] %}
+{% import 'foo' as foo %}
+{% from 'foo' import foo as a, b %}
+
+# SYNTAX 3.3
+# multi-line templating code should be consistent with single-line code
```

Comment:
Perhaps to convey that all four examples below are meant to display consistently as a unit, though only the top two are commented as such at the moment, add instead one line at the top here similar to ``# for each case, all three lines should display as a single unit``.

Reply:
Kinda depends how template markup is handled, for instance the syntax highlighter might wish to highlight keywords or literals differently.

I've updated the comment to make extra clear that whatever the behaviour implemented by the lexer it should be consistent no matter how many lines they are spread over.